app/services/creators/assessment_creator.rb

```diff
@@ -35,7 +35,7 @@ def generate_assessment_hash(remote_ip)
         submission_date: Date.parse(@parsed_raw_post[:submission_date]),
         matter_proceeding_type: @parsed_raw_post[:matter_proceeding_type],
         proceeding_type_codes: ccms_codes_for_application,
-        assessment_type: assessment_type,
+        assessment_type:,
```

Comment:
Out of interest, what does this do?

Reply:
I think this was rubocop (the Ruby linter) which made this change.  It is equivalent to saying `assessment_type: assessment_type`.

Reply:
:wave: it was a new ruby rule.
If the key and value are the same, you can omit the value.
We discussed turning it off as it can make the code more awkward to read but it has stayed in for now.
```ruby
  {
    proceeding_type_codes: ccms_codes_for_application, # value explicitly passed
    assessment_type:, # value not needed because it matches the key name
  }
```

Reply:
Ah just 🪄 things then 😅 
Thanks both!